workers/sentencereviewer/testhelpers/postgres_testcontainer.go

```
@@ -0,0 +1,131 @@
+	return dbConn, teardown
+}
+
+func ClearTestData(db *sql.DB) error {
+	tx, err := db.Begin()
+	if err != nil {
+		return err
+	}
+
+	_, err = tx.Exec("DELETE FROM sentence_reviews")
+	if err != nil {
+		tx.Rollback()
```

golangci-lint (1.64.8)

Error return value of `tx.Rollback` is not checked

(errcheck)